build/root/Makefile

```diff
@@ -309,7 +309,7 @@ clean:
 else
 clean: clean_meta
 	build/make-clean.sh
```

Comment:
it seems a little weird that we have two scripts, each of which cleans a part of the tree.

Reply:
I agree it's not the best, but the build script cleans up all the build specific stuff (_output, but also all the docker images and storage volumes), while the hack/ script is the general "everything else" script.